tests/test_preprocessing/test_returns.py

```diff
@@ -37,3 +37,8 @@ def test_returns(prices, factor_prices):
     assert np.all(X.columns == prices.columns)
     assert np.all(y.columns == factor_prices.columns)
     assert np.all(X.index == y.index)
+
+    # Test index_intersect by making the first column mostly 0's
+    prices[prices.columns.values[0]].iloc[:-10] = 0.0
```

Comment:
to avoid the pandas warning on new version, let's replace by:
```suggestion
    prices.loc[:prices.index[-10], "AAPL"] = 0.0
```